e2e_tests/tests/template/test_template.py

```diff
@@ -306,3 +306,36 @@ def test_rbac_template_exp_create() -> None:
             )
             assert proc.returncode == 1
             assert "not found" in proc.stderr
+
+
+@pytest.mark.e2e_cpu
+def test_get_templates_by_workspace() -> None:
```

Comment:
can you exchange this test for a test like what is in `api_intg_test.go`, where all the other template api tests are.

Reply:
Of course